tests/acceptance/toggle-letter-legal-portrait-landscape-test.js

```diff
@@ -0,0 +1,79 @@
+import { module, test } from 'qunit';
+import {
+  visit,
+  click,
+  fillIn,
+  find,
+} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+import mapboxGlLoaded from '../helpers/mapbox-gl-loaded';
+import mapboxGlDrawReady from '../helpers/mapbox-gl-draw-ready';
+import plotVerticies from '../helpers/plot-verticies';
+
+module('Acceptance | user can toggle between letter/legal and portrait/landscape', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  test('user can toggle between letter/legal and portrait/landscape', async (assert) => {
+    server.createList('project', 10);
+
+    await visit('/projects/new');
```

Comment:
I would go straight to `visit('/projects/10')` and test the workflow from there.